main.tex

```diff
@@ -180,6 +181,117 @@ \section*{Introduction}
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section*{Methods}
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection*{Data collection and processing}
+
+
+%
+For detailed information on Ag1000g WGS sample collection, sequencing, variant calling, quality control and phasing see Ag1000g Consortium \emph{et al}. (\cite{Ag1000gConsortium2017}).
+%
+In brief, \emph{An. gambiae} and \emph{An. coluzzii} mosquitoes were collected from eight countries across Sub-Saharan Africa: Angola, Burkina Faso, Cameroon, Gabon, Guinea, Guinea Bissau, Kenya and Uganda. 
+%
+Guinea Bissau was an \emph{An. gambiae}/\emph{coluzzii} admixed population, from Angola just \emph{An. coluzzii} were sampled, Burkina Faso had samples of both \emph{An. gambiae} and \emph{An. coluzzii} and all other populations consisted of purely \emph{An. gambiae}.
+%
+Mosquitoes were individually whole genome sequenced on the Illumina HiSeq 2000 platform, generating 100bp paired-end reads. 
+%
+Sequenced reads were aligned to the A. gambiae AgamP3 reference genome assembly \cite{Holt2002}). 
+%
+Aligned bam files underwent improvement, before variants were called using GATK UnifiedGenotyper. 
+%
+Quality control included removal of samples with mean coverage <= 14x and an accessibility map was employed following a similar approach to that used for human data by The 1000 Genomes Project Consortium \cite{The1000GenomesProjectConsortium2010}). 
+%
+Variant filtering rules removed accessible variants if: FS < 60.0, HRun > 4, DP <18000 or >32000, MQ < 40, alleles > 2, QD < 5, ReadPosRankSum < -8, overlapped positions repeat masked by DUST (http://blast.wustl.edu/pub/dust). 
+%
+This process produced a call set containing WGS sequences for 765 wild caught individuals which was publicly released \cite{Ag1000gConsortium2017}. 
```

Comment:
I wouldn't bother to repeat details of sample or variant QC. How about:

"Various quality control filters were applied to remove samples and SNPs with poor quality data.

This process produced a call set containing @@N SNPs genotyped in 765 wild-caught individual mosquitoes \cite{Ag1000gConsortium2017}."